e2e/pageobjects/ide/Editor.ts

```diff
@@ -296,6 +298,39 @@ export class Editor {
         return lineYCoordinate;
     }
 
+    async clickOnTheTextInDefinedPosition(line: number, column: number) {
+        const additionalShiftingToY = 19;
+        const additionalShiftingToX = 1;
```

Comment:
The `additionalShiftingToY` and `additionalShiftingToX` constants are used and re-defined in different methods, maybe they should be constants in a class scope.

Reply:
applied